app/consumer.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, args=None, cfg=None):
         super().__init__()
 
     def on_message(self, unused_channel, basic_deliver, properties, body):
-        logger.info('Received message', delivery_tag=basic_deliver.delivery_tag, app_id=properties.app_id)
+        logger.info('Received message', queue=self.QUEUE, delivery_tag=basic_deliver.delivery_tag, app_id=properties.app_id)
```

Comment:
In the process method tx_id gets bound to the logger once decrypted. Can be moved up a few lines though if needed. https://github.com/ONSdigital/sdx-collect/blob/develop/app/response_processor.py#L43